src/Compilers/CSharp/Test/Emit/PDB/PDBLambdaTests.cs

```diff
@@ -1296,6 +1296,156 @@ void F()
 </symbols>");
         }
 
+        [Fact]
+        public void SwitchExpressionWithLambda1()
+        {
+            string source = WithWindowsLineBreaks(@"
+using System;
+
+class C
+{
+    static string M(object o)
+    {
+        return o switch
+        {
+            int i => new Func<string>(() => $""Number: {i} + {o} == {i + (int)o}"")(),
```

Comment:
>$""Number: {i} + {o} == {i + (int)o}"") [](start = 44, length = 39)

Nit: This could be a bit simpler (avoid string interpolations). It's better to simplify tests as much as possible to make it more focused on what's being tested and shorter. In this particular case we are not comparing IL, so it's ok.